damus/Views/Wallet/ConnectWalletView.swift

```diff
@@ -161,6 +162,95 @@ struct ConnectWalletView: View {
                 .padding(2) // Avoids border clipping on the sides
         )
         .padding(.top, 20)
+        .sheet(isPresented: $show_coinos_options, content: {
+            CoinosConnectionOptionsSheet
+        })
+    }
+    
+    var CoinosConnectionOptionsSheet: some View {
+        VStack {
+            Text("How would you like to connect to your Coinos wallet?", comment: "Question for the user when connecting a Coinos wallet.")
+                .font(.title3)
+                .bold()
+                .multilineTextAlignment(.center)
+                .padding(.bottom, 10)
+            
+            Spacer()
+            
+            Button(
+                action: { self.oneClickSetup() },
+                label: {
+                    HStack {
+                        Spacer()
+                        VStack {
+                            HStack {
+                                Image(systemName: "wand.and.sparkles")
+                                Text("One-click setup", comment: "Button label for users to do a one-click Coinos wallet setup.")
+                            }
+                            Text("Also click here if you had a one-click setup before.", comment: "Button description hint for users who may want to do a one-click setup.")
+                                .font(.caption)
+                        }
+                        Spacer()
+                    }
+                }
+            )
+            .frame(maxWidth: .infinity)
+            .buttonStyle(GradientButtonStyle())
+            .opacity(self.userKeypair.privkey == nil ? 0.5 : 1.0)
+            .disabled(self.userKeypair.privkey == nil)
+            
+            if self.userKeypair.privkey == nil {
+                Text("The one-click setup requires the user's private key. Please login with your nsec to use this option.", comment: "Warning text for users who cannot create a Coinos account via the one-click setup without being logged in with their nsec.")
```

Comment:
this message seems a bit scary, it almost sounds like we are sending the users private key to coinos or something. maybe we should make it clear that the users profile is not linked to the coinos account, and there is no identifiable information

Reply:
maybe something like: "you must be logged in with your nsec to use one-click setup. your profile will not be shared with coinos"